common/src/main/java/tachyon/security/UserInformation.java

```diff
@@ -85,6 +85,10 @@ static String getOsPrincipalClassName() {
     return principalClassName;
   }
 
+  static String getOsPrincipalStr() {
```

Comment:
The naming is a bit confusing. Maybe something like `getOsPrincipleClassName` ? Now it feels like you are getting a user string.